src/definition/display.js

```diff
@@ -1,13 +1,18 @@
 module.exports = {
   ipad1536x2048d264: {resolution: {horizontal: 1536, vertical: 2048}, pixelDensity: 264, pixelRatio: 2, orientation: 'portrait'},
   ipad1536x2048d326: {resolution: {horizontal: 1536, vertical: 2048}, pixelDensity: 326, pixelRatio: 2, orientation: 'portrait'},
+  ipad1620x2160d264: {resolution: {horizontal: 1620, vertical: 2160}, pixelDensity: 264, pixelRatio: 2, orientation: 'portrait'},
+  ipad1640x2360d264: {resolution: {horizontal: 1640, vertical: 2360}, pixelDensity: 264, pixelRatio: 2, orientation: 'portrait'},
   ipad1668x2224d264: {resolution: {horizontal: 1668, vertical: 2224}, pixelDensity: 264, pixelRatio: 2, orientation: 'portrait'},
   ipad1668x2388d264: {resolution: {horizontal: 1668, vertical: 2388}, pixelDensity: 264, pixelRatio: 2, orientation: 'portrait'},
   ipad2048x2732d264: {resolution: {horizontal: 2048, vertical: 2732}, pixelDensity: 264, pixelRatio: 2, orientation: 'portrait'},
-  iphone1125x2436: {resolution: {horizontal: 1125, vertical: 2436}, pixelDensity: 458, pixelRatio: 3, orientation: 'portrait'},
-  iphone1242x2208: {resolution: {horizontal: 1242, vertical: 2208}, pixelDensity: 462, pixelRatio: 3, orientation: 'portrait'},
-  iphone1242x2688: {resolution: {horizontal: 1242, vertical: 2688}, pixelDensity: 458, pixelRatio: 3, orientation: 'portrait'},
-  iphone640x1136: {resolution: {horizontal: 640, vertical: 1136}, pixelDensity: 326, pixelRatio: 2, orientation: 'portrait'},
-  iphone750x1334: {resolution: {horizontal: 750, vertical: 1334}, pixelDensity: 326, pixelRatio: 2, orientation: 'portrait'},
-  iphone828x1792: {resolution: {horizontal: 828, vertical: 1792}, pixelDensity: 326, pixelRatio: 2, orientation: 'portrait'},
+  iphone1080x2340d476: {resolution: {horizontal: 1080, vertical: 2340}, pixelDensity: 476, pixelRatio: 3, orientation: 'portrait'},
+  iphone1125x2436d458: {resolution: {horizontal: 1125, vertical: 2436}, pixelDensity: 458, pixelRatio: 3, orientation: 'portrait'},
+  iphone1170x2532d460: {resolution: {horizontal: 1170, vertical: 2532}, pixelDensity: 460, pixelRatio: 3, orientation: 'portrait'},
+  iphone1242x2208d462: {resolution: {horizontal: 1242, vertical: 2208}, pixelDensity: 462, pixelRatio: 3, orientation: 'portrait'},
```

Comment:
See above comments about screen resoloutions. If they are indeed 1920 x 1080, and not 2208 x 1242 then this definition could probably go